README.md

```diff
@@ -1925,6 +1925,25 @@ the list of flags you return from your `FlagsForFile` function in your
 
 See [issue #303][issue-303] for details.
 
+### Install YCM with [NeoBundle][NeoBundle]
+[NeoBundle][NeoBundle] can do the compilation for you just add in your vimrc:
```

Comment:
Change "do the compilation for you just add in your vimrc:" to "do the compilation for you; just add the following to your vimrc:"